README.md

```diff
@@ -54,13 +54,18 @@ The project type and application name can be specified as optional command line
 
 The valid project types are:
 
-- `empty` for Empty Application
+<<<<<<< HEAD
```

Comment:
Yes, I've already spotted this and fixed in other PR to not spoil this:
https://github.com/OmniSharp/generator-aspnet/pull/637